sensor/kubernetes/complianceoperator/handler_impl.go

```diff
@@ -257,7 +257,7 @@ func (m *handlerImpl) processUpdateScanRequest(requestID string, request *centra
 	var updatedScanSettingBinding *unstructured.Unstructured
 	// It is possible that we successfully created the scan setting but had issues on the binding.
 	if ssbObj != nil {
-		updatedScanSettingBinding, err = updateScanSettingBindingFromUpdateRequest(ssObj, request)
+		updatedScanSettingBinding, err = updateScanSettingBindingFromUpdateRequest(ssbObj, request)
```

Comment:
We could either implement a unit test or a full integration test to make sure we're exercising this case.

I'm actually surprised we didn't have one already in https://github.com/stackrox/stackrox/blob/master/tests/compliance_operator_v2_test.go since we test some of the CRUD operations there.

Reply:
Agree. We should add a test in the `compliance_operator_v2_test.go` to cover this path.

Reply:
PR for the e2e test #11866